florist/api/monitoring/metrics.py

```diff
@@ -51,7 +53,14 @@ def add_to_metrics_at_round(self, fl_round: int, data: Dict[str, Any]) -> None:
         self.dump()
 
     def dump(self) -> None:
-        """Dump the current metrics to Redis under the run_id name."""
+        """
```

Comment:
This is beyond the scope of this PR, but I am curious if you think we should explore how to dump metrics to redis at more frequent intervals. If we only do so at the end, then in the case of a crash we lose the metrics. Do you think this is something worthwhile to explore or not really important enough as of now to be thinking about?

Reply:
I did it in this class, I added a call to `dump` at the end of each method so it will update redis every time a new metric is recorded. I think it's overkill for the main class in FL4Health but it was easy enough to instrument that here.